opr-core-testutil/README.md

```diff
@@ -0,0 +1,7 @@
+# Open Product Recovery Core Test Utilities
+
+This package provides skeleton test suites and supporting libraries for testing
+implementations of opr-core interfaces. These packages are intended to ensure
+that all new integrations have the same behavior.
+
+For example, 
```

Comment:
Looks like you might have forgot to finish this thought.

Reply:
Oops! Fixed.